components/atoms/DateModified.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "next-i18next";
+
+export function DateModified(props) {
+  const { t } = useTranslation("common");
+  // TeamCity build dates are received in the format yyyyMMdd
+  const dateFormatted = props.date
+    ? props.date.replace(/^(.{4})(.{2})/gm, "$1-$2-")
+    : "NA";
+  return (
+    <dl className="mt-8 py-2 font-body font-normal text-sm">
+      <dt className="inline">{t("dateModified")}</dt>
+      <dd className="inline">
+        <time>{` ${dateFormatted}`}</time>
```

Comment:
I know Canada.ca doesn't do it, but let's put in a `datetime` property, as documented on MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
Added datetime property